lms/static/scripts/postmessage_json_rpc/server/methods.js

```diff
@@ -9,8 +9,37 @@
 /**
  * Return a Hypothesis client config object for the current LTI request.
  */
-export function requestConfig() {
+export async function requestConfig() {
   const configEl = document.querySelector('.js-config');
   const clientConfigObj = JSON.parse(configEl.textContent).hypothesisClient;
-  return clientConfigObj;
+  return Promise.resolve(clientConfigObj);
```

Comment:
don't need to return a resolved promise, async already does that. Just return the result.